backend/src/exploregens.ts

```diff
@@ -99,7 +88,11 @@ export class ExploreGens {
                 await this.updateAllInstalledGenerators();
             }
         }
-    }
+	}
+	
+	private isInTheia(): boolean {
+		return this.inTheia;
+	}
```

Comment:
no need for this func. can directly use this.isInTheia member.

Reply:
There is already method with this name

Reply:
This method is used in UI through rpc

Reply:
why there's a need for this rpc func. you are passing the boolean isInTheia to exolregens constructor.